airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -69,6 +69,7 @@ class Meta:
     tags = fields.List(fields.Nested(DagTagSchema), dump_only=True)
     max_active_tasks = auto_field(dump_only=True)
     max_active_runs = auto_field(dump_only=True)
+    max_consecutive_failed_dag_runs = auto_field(dump_only=True)
```

Comment:
I wonder about the name. `max_consecutive_failed_dag_runs` might be interpreted as not allowing scheduling after the X failures but this is not really a hard limit - user can override it by enabling the dag. For the other max properties user can't really override them.

Reply:
@eladkal,
with current logic,
If a user wants they can manually enable the dag, but its again failed, then the dag will automatically be disabled,
as we are counting last N failed which will again be true.

Reply:
for naming, 
yeah its a good point.
happy to rename it , please suggest if we can come up with better naming for messaging.

Reply:
I need to think about the name.

> if user wants to restart counting again, they will have add to add a non-failed dag_run for now
That might be problematic however I think we can avoid deciding right now.
We can mark this feature as experimintal thus it allows the flexability of changing the behavior if needed.

Reply:
I thought a bit about this and what about: `max_allowed_dag_failures_in_row` or just `max_dag_failures_in_row`?